debian/copyright

```diff
@@ -0,0 +1,9 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ev3dev-lang-python
```

Comment:
It was not too long ago that I had never done any debian packaging either. :blush: